build/rust/config.gni

```diff
@@ -28,6 +28,7 @@ cargo_lib_prefix = ""
 cargo_lib_suffix = ""
 
 default_rustflags = [ "-Coverflow-checks=on" ]
+default_rustflags += [ "-Cpanic=abort" ]
```

Comment:
Is the setting in `build/rust/Cargo.toml` not sufficient?

Reply:
yes, but this doesn't hurt

Reply:
Ok.